maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/mapper/ApplicationTrackingMapper.java

```diff
@@ -102,5 +128,11 @@ protected Hardship buildHardship(WorkflowRequest workflowRequest) {
                 .withHardshipType(HardshipType.CCHARDSHIP);
     }
 
-
+    public String getCapitalisedFullName(String... names) {
+        return Arrays.stream(names)
+                .map(StringUtils::trim)
+                .filter(StringUtils::isNotBlank)
```

Comment:
We can move this to commonsUtil?

Reply:
Yes we can.. Should we move this whenever it is going to be re-used rather than now?